integration-tests/functions/src/api/i-am-typescript.ts

```diff
@@ -0,0 +1,3 @@
+export default function topLevel(req, res) {
```

Comment:
We should add proper TS types here to the function so that it's a realistic TS file and we e.g. see that type errors would show or something

Reply:
heh, as it turned out typescript hadn't been setup so good call 👍 fixed in https://github.com/gatsbyjs/gatsby/pull/30192/commits/af36e0960c05d225affc3cf98db6a02fec87100c